From 986345ae88110a7c0810d608e87271be369fb218 Mon Sep 17 00:00:00 2001
From: Sergey Fedorov <barracuda@macos-powerpc.org>
Date: Mon, 21 Apr 2025 20:53:20 +0800
Subject: [PATCH] Revert "osx/core/private.h: fix parsing included CarbonCore
 header on powerpc"

This reverts commit c3141dfe3e065c6913f9c3655a649240375f408d.
While it worked to fix compilation, it did not address the cause of the issue,
and was a suboptimal solution. To avoid parsing error on powerpc, disable
precompiled headers usage in configure args.
---
 include/wx/osx/core/private.h | 6 ------
 1 file changed, 6 deletions(-)

diff --git a/include/wx/osx/core/private.h b/include/wx/osx/core/private.h
index 18f1d34d13..a771611296 100644
--- include/wx/osx/core/private.h
+++ include/wx/osx/core/private.h
@@ -12,12 +12,6 @@
 #ifndef _WX_PRIVATE_CORE_H_
 #define _WX_PRIVATE_CORE_H_
 
-// Without this MachineExceptions.h of CarbonCore cannot be parsed
-// when C++11 or higher standard is used.
-#ifdef __APPLE_ALTIVEC__
-#undef __APPLE_ALTIVEC__
-#endif
-
 #include "wx/defs.h"
 
 #include <CoreFoundation/CoreFoundation.h>
